test/UpdatedCommand.js

```diff
@@ -353,6 +353,45 @@ describe("UpdatedCommand", () => {
       updatedCommand.runCommand(exitWithCode(1, done));
     });
   });
+
+  /** =========================================================================
+   * JSON Output
+   * ======================================================================= */
+
+  describe("with --json", () => {
+    let testDir;
+
+    beforeEach(() => initFixture("UpdatedCommand/basic").then((dir) => {
+      testDir = dir;
+    }));
+
+    it("should list changes as a json object", (done) => {
+      setupGitChanges(testDir, [
+        "packages/package-2/random-file",
+      ]);
+
+      const updatedCommand = new UpdatedCommand([], {
+        json: true
+      }, testDir);
+
+      updatedCommand.runValidations();
+      updatedCommand.runPreparations();
+
+      updatedCommand.runCommand(exitWithCode(0, (err) => {
+        if (err) return done.fail(err);
+        try {
+          const outputStr = consoleOutput();
+          expect(outputStr).toMatchSnapshot();
```

Comment:
I think we can just skip straight to the `JSON.parse(consoleOutput())` snapshot, as we really shouldn't be testing whether or not `JSON.stringify()` is working correctly.